tutorials/circuits/3_summary_of_quantum_operations.ipynb

```diff
@@ -384,12 +312,12 @@
    "source": [
     "### Identity gate\n",
     "\n",
-    "The identity gate is $Id = u1(0)$."
+    "The identity gate is $Id = p(0)$."
```

Comment:
For some reasons this equation has "Math processing error" on ReviewNB webiste. 
<img width="302" alt="image" src="https://user-images.githubusercontent.com/7631333/171163293-4903b9e8-859f-4853-b477-0c1a35a17e7a.png">

Reply:
It's rendering correctly when openning on my local computer. We can ignore it for now. I will check again once this is merged.